tests/cases/gts/component-class-one-line.gts

```diff
@@ -0,0 +1,18 @@
+import Component from '@glimmer/component';
+import { ComponentLike } from '@glint/template';
+
+interface Signature {
+  Args: {
+    component: ComponentLike<never>;
+  };
+}
+
+const half = (x: number) => x / 2;
+
+export default class Compiler extends Component<Signature> {
+  <template>{{yield (hash component=this.component) half=half}}</template>
```

Comment:
I added a minimal test case for the component-class one-liner bug here: https://github.com/gitKrystan/prettier-plugin-ember-template-tag/pull/52/files